### ISSUE TITLE
Unable to generate a URL for the named route "sonata_user_admin_resetting_request" as such route does not exist.

### ISSUE THREAD

**Issue by Azouz-Jribi** [sonata-project/SonataUserBundle#702]
```
"friendsofsymfony/user-bundle": "2.0.x-dev#6ccff96",
"sonata-project/user-bundle": "dev-add_support_for_fos_user2"
```
# routing.yml

``` yaml
sonata_user:
    resource: '@SonataUserBundle/Resources/config/routing/admin_security.xml'
    prefix: /admin

sonata_user_security:
    resource: "@SonataUserBundle/Resources/config/routing/sonata_security_2.xml"

sonata_user_resetting:
    resource: "@SonataUserBundle/Resources/config/routing/sonata_resetting_2.xml"
    prefix: /resetting

sonata_user_profile:
    resource: "@SonataUserBundle/Resources/config/routing/sonata_profile_2.xml"
    prefix: /profile

sonata_user_register:
    resource: "@SonataUserBundle/Resources/config/routing/sonata_registration_2.xml"
    prefix: /register

sonata_user_change_password:
    resource: "@SonataUserBundle/Resources/config/routing/sonata_change_password_2.xml"
    prefix: /profile
```

**Comment by marvoh** [comment]
Did you find a solution for this?

**Comment by Bramboz** [comment]
add this in your app/config/routing.yml, according with point 2.3.3 of the doc (https://sonata-project.org/bundles/user/master/doc/reference/installation.html):

sonata_user_admin_security:
    resource: '@SonataUserBundle/Resources/config/routing/admin_security.xml'
    prefix: /admin

sonata_user_admin_resetting:
    resource: '@SonataUserBundle/Resources/config/routing/admin_resetting.xml'
    prefix: /admin/resetting

**Comment by OskarStark** [comment]
Closed via #869
